README.md

```diff
@@ -56,44 +53,30 @@ A simplified manifest would look like this:
   "name": "My Company",
   "implementation": "My implementation",
   "issuers": [{
-    "id": "",
-    "endpoint": "https://mycompany.example/credentials/issue",
-    "method": "POST",
-    "supportedEcdsaKeyTypes": ["P-256", "P-384"]
-    "tags": ["ecdsa-rdfc-2019"]
-  }, {
-    "id": "",
-    "endpoint": "https://mycompany.example/credentials/issue",
-    "method": "POST",
-    "supportedEcdsaKeyTypes": ["P-256"]
-    "tags": ["ecdsa-jcs-2019"]
-  }, {
     "id": "",
     "endpoint": "https://mycompany.example/credentials/issue",
     "method": "POST",
```

Comment:
We just removed the `method` property in other implementation descriptions. Is it needed here?